tests/new_character_test.cpp

```diff
@@ -145,7 +145,10 @@ TEST_CASE( "starting_items" )
                     g->u.male = i == 0;
                     std::list<item> items = prof->items( g->u.male, traits );
                     for( const item &it : items ) {
-                        items.insert( items.begin(), it.contents.begin(), it.contents.end() );
+                        const std::list<const item *> it_contents = it.contents.all_items_top();
+                        for( const item *top_content_item : it_contents ) {
+                            items.push_back( item( *top_content_item ) );
```

Comment:
```suggestion
                            items.push_back( *top_content_item );
```
`push_back` already copies the item, no need to copy a copy of it.